src/battle_main.c

```diff
@@ -3316,7 +3317,7 @@ const u8* FaintClearSetData(u32 battler)
 
     for (i = 0; i < gBattlersCount; i++)
     {
-        if (i != battler && GetBattlerSide(i) != GetBattlerSide(battler))
```

Comment:
Minor nitpick but maybe revert this change? I feel like having the function call makes it more clear what is happening and it isn't used anywhere else.